### PR TITLE
Typos from WordPress history

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

**Comment by larsoner** [comment]
Needs merge/rebase

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2142,6 +2172,7 @@ busness->business
 buss->bus
 busses->buses
 bussiness->business
+buttom->button
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "bottom"

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -2391,6 +2427,7 @@ caterpiller->caterpillar
 caterpillers->caterpillars
 cathlic->catholic
 catholocism->catholicism
+catlink->taglink
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: This seems not to be obvious.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -4845,6 +4904,7 @@ dialgo->dialog
 dialgos->dialogs
 dialig->dialog
 dialigs->dialogs
+dialoge->dialog
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "dialogue" (BE)

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -5508,6 +5575,7 @@ ecidious->deciduous
 eclispe->eclipse
 ecognized->recognized
 ecomonic->economic
+ecret->secret
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "erect"

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -5922,12 +5995,14 @@ equallity->equality
...
 equel->equal
 equialent->equivalent
+equilavalent->equivalent
 equilibium->equilibrium
 equilibrum->equilibrium
 equiped->equipped
 equipmentd->equipment
 equipments->equipment
 equippment->equipment
+equire->require
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "enquire"

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -8861,6 +8961,7 @@ jouney->journey
 journied->journeyed
 journies->journeys
 joystik->joystick
+jpng->png
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "jpg"

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -9435,6 +9541,7 @@ meanining->meaning
 meaninless->meaningless
 meaninng->meaning
 mear->wear, mere, mare,
+mearly->merely
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "nearly"

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -10168,6 +10285,7 @@ notication->notification
 notications->notifications
 noticeing->noticing
 noticible->noticeable
+notidy->notify
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "not tidy"

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -10899,10 +11025,12 @@ paratmers->parameters
 parctically->practically
 pard->part
 parellels->parallels
+parem->param
 paremeter->parameter
 paremeters->parameters
 paremter->parameter
 paremters->parameters
+parentesis->parentheses
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also singular "parenthesis"

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -11699,13 +11832,15 @@ principlaity->principality
 principly->principally
 princliple->principle
 prind->print
+pring->print
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "bring", "ping"

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -12360,6 +12504,7 @@ rectange->rectangle
 rectanges->rectangles
 rectanglar->rectangular
 rectiinear->rectilinear
+rects->rect
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also programming slang for "rectangles"

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -13301,6 +13453,9 @@ samller->smaller
 sanctionning->sanctioning
 sandwhich->sandwich
 Sanhedrim->Sanhedrin
+sanitiation->sanitisation
+sanitised->sanitized
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: AE <-> BE conversion

File: codespell_lib/data/dictionary.txt [hunk 13]
```diff
@@ -14348,6 +14520,7 @@ sturctures->structures
 sturucture->structure
 stutus->status
 styhe->style
+stypes->styles
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "styles"

File: codespell_lib/data/dictionary.txt [hunk 14]
```diff
@@ -15270,6 +15454,7 @@ tomorrrow->tomorrow
 tongiht->tonight
 tood->todo
 toogle->toggle
+tooglecomparetwocheckbox->togglecomparetwocheckbox
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: very specific

File: codespell_lib/data/dictionary.txt [hunk 15]
```diff
@@ -15312,6 +15498,7 @@ trafficed->trafficked
 trafficing->trafficking
 trafic->traffic
 traiing->trailing, training,
+trailingshashit->trailingslashit
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: very specific

File: codespell_lib/data/dictionary.txt [hunk 16]
```diff
@@ -15584,6 +15773,7 @@ unabale->unable
 unabel->unable
 unacceptible->unacceptable
 unaccesible->unaccessible
+unaccurate->inaccurate
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: Obsolete, but not completely wrong. Maybe offer both variants.
https://en.wiktionary.org/wiki/unaccurate

File: codespell_lib/data/dictionary.txt [hunk 17]
```diff
@@ -16051,6 +16247,7 @@ uptim->uptime
 uptions->options
 upto->up to
 uptream->upstream
+uris->urls
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: URI is as good as URL
https://en.wikipedia.org/wiki/Uniform_Resource_Identifier

File: codespell_lib/data/dictionary.txt [hunk 18]
```diff
@@ -16686,6 +16898,7 @@ yhe->the
 yieldin->yielding
 ymbols->symbols
 yotube->youtube
+youn->your
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: also "you", "young"

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 8 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done.

File: codespell_lib/data/dictionary.txt (hunk 9 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 10 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 11 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 12 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: I don't understand

File: codespell_lib/data/dictionary.txt (hunk 13 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Do you mean add singular form? Done

File: codespell_lib/data/dictionary.txt (hunk 14 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 15 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 16 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 17 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 18 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

**Comment by Gelma** [comment]
@larsoner @peternewman @luzpaz 

This also should be fixed. Please check it out, so we can merge it.

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Also equine and esquire

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Also jpeg

File: codespell_lib/data/dictionary.txt (hunk 11 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: This says it's only an abbreviation, remove it? https://www.collinsdictionary.com/dictionary/english/rect

File: codespell_lib/data/dictionary.txt (hunk 12 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: American English/British English, remove it and stick a note in #103 .

File: codespell_lib/data/dictionary.txt (hunk 13 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Also types

File: codespell_lib/data/dictionary.txt (hunk 16 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: As per https://github.com/codespell-project/codespell/issues/712 I really don't think we should have real words in the dictionary, add it to the list for that issue.

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed.

File: codespell_lib/data/dictionary.txt (hunk 13 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 12 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

File: codespell_lib/data/dictionary.txt (hunk 11 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Added

**Comment by Gelma** [comment]
I'm going to push new fixes

File: codespell_lib/data/dictionary.txt [hunk 19]
```diff
@@ -9084,6 +9173,7 @@ hel->help, hell, heal,
 helerps->helpers
 hellow->hello
 helment->helmet
+helo->ehlo
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: please remove

File: codespell_lib/data/dictionary.txt [hunk 20]
```diff
@@ -9374,6 +9466,7 @@ idices->indices
 idiosyncracies->idiosyncrasies
 idiosyncracy->idiosyncrasy
 idividual->individual
+idpost->post
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: remove

File: codespell_lib/data/dictionary.txt [hunk 21]
```diff
@@ -13551,6 +13680,7 @@ placeholde->placeholder
 placeholdes->placeholders
 placeholer->placeholder
 placeholers->placeholders
+placematt->placemat
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: maybe add `placement` to?

File: codespell_lib/data/dictionary.txt [hunk 22]
```diff
@@ -15326,6 +15469,8 @@ remoote->remote
 remore->remote
 remorted->reported
 removeable->removable
+removedchild->removechild
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: `removedchild` sounds valid to me

File: codespell_lib/data/dictionary.txt [hunk 23]
```diff
@@ -16443,6 +16598,7 @@ shortcomming->shortcoming
 shortcommings->shortcomings
 shortend->shortened, short end,
 shortern->shorten
+shortkuts->shortcuts
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: i guess we can also add `shortkut->shortcut` while we're at it.

File: codespell_lib/data/dictionary.txt [hunk 24]
```diff
@@ -16536,6 +16693,7 @@ signular->singular
 signularity->singularity
 silentely->silently
 silenty->silently
+siliently->silently
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: please add `saliently` as a secondary result

File: codespell_lib/data/dictionary.txt [hunk 25]
```diff
@@ -19140,6 +19325,7 @@ unrestrcited->unrestricted
 unroated->unrotated
 unsable->unusable, usable, unstable,
 unsccessful->unsuccessful
+unschedule->unschedules
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: `unschedule` is a valid word. Please remove.

File: codespell_lib/data/dictionary.txt [hunk 26]
```diff
@@ -19164,7 +19352,6 @@ unsual->unusual
 unsubscibe->unsubscribe
 unsubscibed->unsubscribed
 unsubscibing->unsubscribing
-unsubscritpion->unsubscription
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: please re-add this entry

File: codespell_lib/data/dictionary.txt [hunk 27]
```diff
@@ -19303,6 +19491,7 @@ usefullness->usefulness
 usefult->useful
 usefuly->usefully
 usefutl->useful
+useg->user
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: also `usage`

File: codespell_lib/data/dictionary.txt [hunk 28]
```diff
@@ -19338,6 +19527,7 @@ usueful->useful
 ususal->usual
 ususally->usually
 UTF8ness->UTF-8-ness
+uti->uri
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: i worry about 3 letter typos like this that will trigger false positives. Maybe designate it as a warning instead?

File: codespell_lib/data/dictionary.txt [hunk 29]
```diff
@@ -19778,10 +19973,13 @@ whitepsaces->whitespaces
 whith->with
 whithin->within
 whithout->without
+whitout->without
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: also `whiteout`

File: codespell_lib/data/dictionary.txt [hunk 30]
```diff
@@ -19896,7 +20094,9 @@ wonderfull->wonderful
 wont't->won't
 wont->won't, wont,
 worarounds->workarounds
+wordcamps->wordcamp
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: `wordcamps` is a plural of `wordcamp` right?

File: codespell_lib/data/dictionary.txt [hunk 31]
```diff
@@ -19934,6 +20134,8 @@ wouldn;t->wouldn't
 wouldnt'->wouldn't
 wouldnt->wouldn't
 wouldnt;->wouldn't
+wpb->wpdb
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: not sure if we should add this.

**Comment by luzpaz** [comment]
Please rebase (and also please don't add anymore commits to this PR). Thanks!

File: codespell_lib/data/dictionary.txt (hunk 31 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 23 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Added

File: codespell_lib/data/dictionary.txt (hunk 24 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Added

File: codespell_lib/data/dictionary.txt (hunk 25 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

File: codespell_lib/data/dictionary.txt (hunk 26 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Re-added

File: codespell_lib/data/dictionary.txt (hunk 27 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Added

File: codespell_lib/data/dictionary.txt (hunk 28 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 29 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 30 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

File: codespell_lib/data/dictionary.txt (hunk 19 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: remved

File: codespell_lib/data/dictionary.txt (hunk 20 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 21 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

File: codespell_lib/data/dictionary.txt (hunk 22 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

File: codespell_lib/data/dictionary.txt [hunk 32]
```diff
@@ -7410,6 +7484,7 @@ errro->error
 errror->error
 errrors->errors
 errros->errors
+errstr->errorstr
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt [hunk 33]
```diff
@@ -8236,6 +8318,7 @@ finxing->fixing
 fiorget->forget
 firdt->first, third,
 firends->friends
+firest->fires
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 34]
```diff
@@ -6846,6 +6911,7 @@ easer->easier, eraser,
 easili->easily
 easilly->easily
 easyer->easier
+eaturn->return
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Comment by Gelma** [comment]
> Please rebase (and also please don't add anymore commits to this PR). Thanks!

I'm going to upload fixes

File: codespell_lib/data/dictionary.txt (hunk 34 above)
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: @Gelma where is this fix?

File: codespell_lib/data/dictionary.txt (hunk 20 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: wasn't removed

File: codespell_lib/data/dictionary.txt (hunk 33 above)
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: wasn't removed

File: codespell_lib/data/dictionary.txt (hunk 32 above)
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: wasn't removed

File: codespell_lib/data/dictionary.txt (hunk 21 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: wasn't removed (which the request wasn't for) or changed.

**Comment by Gelma** [comment]
> @Gelma it's confusing when you write you've done something and then haven't especially with PRs this big because we can lose track of requested changes and merge which pollutes the `dictionary.txt` file

Found what happened. I have different branches. I made fix in both branches (main and pushed), mixing things up. Sorry, I'm going to update the missing part.

**Comment by luzpaz** [comment]
Yea, this is the reason why I don't submit enormous PRs like I've done in the past. It's too much of a hassle to keep track of, fix, rebase, listen to feedback and revise..etc..etc..

**Comment by Gelma** [comment]
> Yea, this is the reason why I don't submit enormous PRs like I've done in the past. It's too much of a hassle to keep track of, fix, rebase, listen to feedback and revise..etc..etc..

Well, I'm going to merge the missed part.

**Comment by luzpaz** [comment]
LGTM

**Comment by luzpaz** [comment]
This ticket can be closed in favor of merged PRs #1231 #1232 #1233  
Thanks @Gelma !

**Pull request closed without merging**